samples/drumthumper/src/main/cpp/DrumPlayerJNI.cpp

```diff
@@ -79,13 +80,19 @@ JNIEXPORT void JNICALL Java_com_plausiblesoftware_drumthumper_DrumPlayer_loadWav
     WavStreamReader reader(&stream);
     reader.parse();
 
+    int sampleRate = reader.getSampleRate();
+    int numChannels = reader.getNumChannels();
+    jboolean isFormatValid = sampleRate == 41000 && numChannels == 1;
```

Comment:
Somewhere there is probably a 44100 used when opening the stream.
This code and that code should share a constant, maybe kRequiredSampleRate.
That will make it easier for people to replace the 44100 Hz samples with another rate.
Same for numChannels.

Reply:
Ack. Done.